modules/RTC/TraceablePeerConnection.js

```diff
@@ -568,26 +572,23 @@ TraceablePeerConnection.prototype.addStream = function (stream, ssrcInfo) {
     if (stream)
         this.peerconnection.addStream(stream);
     if (ssrcInfo && ssrcInfo.type === "addMuted") {
-        this.sdpConsistency.setPrimarySsrc(ssrcInfo.ssrc.ssrcs[0]);
+        this.sdpConsistency.setPrimarySsrc(ssrcInfo.ssrcs[0]);
         const simGroup =
-            ssrcInfo.ssrc.groups.find(groupInfo => {
-                return groupInfo.group.semantics === "SIM";
+            ssrcInfo.groups.find(groupInfo => {
+                return groupInfo.semantics === "SIM";
             });
         if (simGroup) {
-            const simSsrcs = SDPUtil.parseGroupSsrcs(simGroup.group);
-            this.simulcast.setSsrcCache(simSsrcs);
+            this.simulcast.setSsrcCache(simGroup.ssrcs);
         }
         const fidGroups =
-            ssrcInfo.ssrc.groups.filter(groupInfo => {
-                return groupInfo.group.semantics === "FID";
+            ssrcInfo.groups.filter(groupInfo => {
```

Comment:
i have a feeling that i had some debug info here because this could be shorter, feel free to change to this (if it won't cause conflicts, if so no big deal for now):

`ssrcInfo.groups.filter(groupInfo => groupInfo.semantics === "FID"`